src/components/Text/Text.tsx

```diff
@@ -1,14 +1,24 @@
 import * as React from 'react';
 import { st, classes } from './Text.st.css';
-import { DEFAULT_TAG_NAME, TYPOGRAPHY } from './constants';
+import { DEFAULT_TAG_NAME, TYPOGRAPHY, TEXT_PRIORITY } from './constants';
```

Comment:
There is no need for the `Text` prefix since it is known that it is a priority for the text.
Let's change it for PRIORITY.

Reply:
there is already a PRIORITY enum used in <Button /> component, so I had to name this TEXT_PRIORITY to avoid collision as they're both exported in components/index.ts

Reply:
Nice catch! Ok so keep this name:)